<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Android Photos Archive</string>
  </property>
  <widget class="QWidget" name="mentralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QProgressBar" name="mProgressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mPushButtonPrevious">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mPushButtonNext">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="mImageLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mMenubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="separator"/>
    <addaction name="ActionInfo"/>
    <addaction name="separator"/>
    <addaction name="ActionCloseEvent"/>
   </widget>
   <widget class="QMenu" name="menuUstawienia">
    <property name="title">
     <string>Ustawienia</string>
    </property>
    <addaction name="ActionDataBaseConnection"/>
   </widget>
   <widget class="QMenu" name="menuSerwer">
    <property name="title">
     <string>Serwer</string>
    </property>
    <addaction name="ActionRunServer"/>
    <addaction name="ActionStopServer"/>
    <addaction name="separator"/>
    <addaction name="ActionServerSettings"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuSerwer"/>
   <addaction name="menuUstawienia"/>
  </widget>
  <widget class="QStatusBar" name="mStatusbar"/>
  <widget class="QToolBar" name="mToolBarServer">
   <property name="windowTitle">
    <string>Pasek narzędzi serwer</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionRunServer"/>
   <addaction name="ActionStopServer"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionInfo"/>
  </widget>
  <action name="ActionRunServer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start serwer</string>
   </property>
   <property name="toolTip">
    <string>Przełącz serwer w tryb nasłuchiwania</string>
   </property>
   <property name="shortcut">
    <string>Shift+N</string>
   </property>
  </action>
  <action name="ActionCloseEvent">
   <property name="text">
    <string>Zamknij </string>
   </property>
   <property name="toolTip">
    <string>Zamknij program</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="ActionDataBaseConnection">
   <property name="text">
    <string>Połączenie z bazą danych </string>
   </property>
   <property name="toolTip">
    <string>Ustawienie połączenia z bazą danych</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="ActionServerSettings">
   <property name="text">
    <string>Ustawienia</string>
   </property>
   <property name="toolTip">
    <string>Ustawienia komunikacji i portu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="ActionStopServer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop serwer</string>
   </property>
   <property name="toolTip">
    <string>Zastopuj serwer. Przerwij nasłuchiwanie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="ActionInfo">
   <property name="text">
    <string>O programie</string>
   </property>
   <property name="toolTip">
    <string>Informacje o programie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ActionCloseEvent</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
